supertylerc/died
# Closes #1. Fix Travis CI builds.

Changes:

- Remove static ruby version from `Gemfile`
+ Add `vendor/**` to exclusion path for rubocop
- Remove ruby 1.9.3 (foodcritic requires >=2.0.0)
* Change paths for `yamllint` and `jsonlint` in `Rakefile`
* Fix status codes from shell commands
* Rename `cookbooks/files/.tmux.conf` to `cookbooks/files/tmux.conf`

Caveats:

* None

Test Plan:

* Verify Travis CI no longer complains of `Gemfile` ruby version
* Verify Travis CI no longer tries to lint `vendor/`
* Verify `foodcritic` gets installed in Travis CI
* Verify there are no errors for `yamllint`
* Verify there are no errors for `jsonlint`
